dashboard/streamlit_app.py

```diff
@@ -38,8 +38,44 @@ def extract_id_from_string(string: str) -> int:
     return None
 
 
+# ========== FUNCTIONS: METRICS ==========
+def get_metric_judge(conn: connect):
```

Comment:
Another great function. This just adds some nice visual metrics that a user can better understand the data at hand.